plone/app/layout/viewlets/content.py

```diff
@@ -71,43 +71,7 @@ def show(self):
             ISecuritySchema,
             prefix='plone',
         )
-        return not self.anonymous or settings.allow_anon_views_about
```

Comment:
Why not showing for non-anonymous too?
Only reason I can think of is, that the some of the information is available in the toolbar too. But is the toolbar also visible for simple members without editing rights? Then we need this viewlet also for them.